### DISCUSSION TITLE
How to insert -amd in maven build ?

### DISCUSSION THREAD

**Discussion by r1gm** [apache/netbeans#7853] [Q&A]
Hi,

How i insert -amd command in maven build ?

**Comment by mbien** [comment]
project properties -> actions -> build project -> check "build with dependencies" should set this flag I believe

**Reply by r1gm** [reply]
Hello,

i already tried that flag and it's working just like i need to, i just curious is it had same behavior with -amd ?

and why if i choose "Build Recursively (With Modules)" and press "OK" the option is not available at nbactions.xml ?
